### PR TITLE
fix:changes the type of children to ReactText

### PR DESCRIPTION
**Pull request by stutirao**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/legend/Legend.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ export const Legend = (props: LegendProps) => {
         appearance={labelAppearance}
         weight={labelWeight}
       >
-        {children}
+        {children.toString()}
```
Comment (aditya-kumawat) [inline core/components/atoms/legend/Legend.tsx]: Why is this required?

**Review by satyamyadav — APPROVED** [review]
